packages/yourloops/pages/login/verify-email-page.tsx

```diff
@@ -25,67 +25,172 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React, { type FunctionComponent, type MouseEventHandler } from 'react'
+import React, { type FunctionComponent } from 'react'
 import Button from '@mui/material/Button'
-import { useTranslation } from 'react-i18next'
+import { Trans, useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import { useAuth0 } from '@auth0/auth0-react'
-import Divider from '@mui/material/Divider'
 import Container from '@mui/material/Container'
 import config from '../../lib/config/config'
 import { GlobalStyles } from 'tss-react'
-import { useTheme } from '@mui/material'
+import Link from '@mui/material/Link'
+import { useAuth } from '../../lib/auth'
+import { useAlert } from '../../components/utils/snackbar'
+import { AUTH0_ERROR_EMAIL_NOT_VERIFIED } from '../../lib/auth/models/auth0-error.model'
+import AppBar from '@mui/material/AppBar'
+import Toolbar from '@mui/material/Toolbar'
+import Avatar from '@mui/material/Avatar'
+import { makeStyles } from 'tss-react/mui'
+import { type Theme } from '@mui/material/styles'
+
+const classes = makeStyles()((theme: Theme) => ({
+  appBar: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    zIndex: theme.zIndex.drawer + 1,
+    backgroundColor: theme.palette.common.white,
+    color: 'var(--text-color-primary)'
+  },
+  desktopLogo: {
+    width: 140
+  }
+}))
 
 const VerifyEmailPage: FunctionComponent = () => {
-  const { loginWithRedirect, logout } = useAuth0()
+  const { classes: { appBar, desktopLogo } } = classes()
+  const { loginWithRedirect, getAccessTokenSilently } = useAuth0()
   const { t } = useTranslation()
-  const theme = useTheme()
+  const { logout: authLogout } = useAuth()
+  const alert = useAlert()
+
+  const sendConfirmationEmail = async (): Promise<void> => {
```

Comment:
I suggest a more approriate function name that precise we are contacting the support (and not sending a confirmation email). Maybe `contactCustomerSupport` ?